docsrc/conf.py

```diff
@@ -539,7 +539,7 @@
         'cyrus-stable': ('https://www.cyrusimap.org%s',None),
         'github-release':
             ('https://github.com/cyrusimap/cyrus-imapd/releases/tag/cyrus-imapd-%s',
-             'cyrus-imapd-'),
+             'cyrus-imapd-%s'),
```

Comment:
Same comment as the earlier '%s' changes.  Please just drop this commit from your PR.